### PR TITLE
Update known bugs in HTCondor-CE

### PR DESCRIPTION
**Pull request by brianhlin**
Noticed this while reviewing OSG 3.6 update instructions

### CODE REVIEW

**Review by timtheisen — COMMENTED** [review]
When do you want this update to go out? The HTCondor in 3.6-upcoming still trips over C style comments. If this goes out next Tuesday, fine. Otherwise, we need both known issues.

**Comment by brianhlin** [comment]
Oh duh, good catch! Reverted and added a small clarification

**Review by timtheisen — APPROVED** [review]

File: docs/release/updating-to-osg-36.md [hunk 1]
```diff
@@ -219,8 +219,14 @@ To update OSG-Configure, perform the following steps:
 
 #### HTCondor-CE ####
 
+!!! bug "Passing along non-HTCondor batch system directives"
+    `default_CERequirements` in the the new Job Router ClassAd transform syntax is ignored.
+    To fix this, apply the change in [this patch](https://github.com/htcondor/htcondor-ce/pull/530/files) to
+    `/usr/share/condor-ce/config.d/01-ce-router-defaults.conf`.
+    The next release of HTCondor-CE will contain this fix and will not require any additional action post-update.
```
Comment (matyasselmeci) [inline docs/release/updating-to-osg-36.md:226]: What release is the "next release" ?

**Review by matyasselmeci — APPROVED** [review]
One comment, otherwise it's fine.

**Pull request merged**